rstudio/rstudio.sh

```diff
@@ -15,7 +15,19 @@
 # This init script installs RStudio Server on the master node of a Cloud
 # Dataproc cluster.
 
-set -x -e
+set -euxo pipefail
+
+# Only run on the master node
+ROLE="$(/usr/share/google/get_metadata_value attributes/dataproc-role)"
+
+USER_NAME="$(/usr/share/google/get_metadata_value attributes/rstudio-user || echo rstudio)"
+USER_PASSWORD="$(/usr/share/google/get_metadata_value attributes/rstudio-password || true)"
+
+RSTUDIO_SERVER_VERSION=1.2.5019
+RSTUDIO_SERVER_PACKAGE=rstudio-server-${RSTUDIO_SERVER_VERSION}-amd64.deb
```

Comment:
Seems the package name pattern is different between ubuntu and debian. This is why in the existing code, they use if/else.

Reply:
Maybe it was different in the past, but not now:
https://rstudio.com/products/rstudio/download-server/debian-ubuntu/